projects/packages/search/src/dashboard/components/dashboard/search-promotion.scss

```diff
@@ -0,0 +1,16 @@
+.jp-search-dashboard-promotion {
+	h1 {
+		font-size: var( --font-title-large );
+		font-weight: 600;
+		line-height: 1.2;
+	}
+
+	ul.jp-product-promote li {
+		background: url( data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAMAAABg3Am1AAAANlBMVEVHcEwFnwUInggGnggGnggHnAcAnwUFnQcAnwcGnwkFnQgGnQgFnwcGnQYFnQcFnAcGnQkDnwdhiL0pAAAAEnRSTlMAMF//f2Aw7yBQ3+9gcIBgcED+HDbkAAAAZklEQVR4Ae3LNwICARDDQC0+cv7/Y8mwV9odSfWIcf/+VegnGkIvDaGXKvTTn/Gz+Uf5xTL0K1XotS7fs5H6GHvvaO8d7c3j7rdgHne/A/PYt/cO+R42oYdN6OEQetiFHo4A//6dAXqtBEkmtWutAAAAAElFTkSuQmCC )
```

Comment:
Can we add a comment saying what this image is?

Reply:
Done!